### PR TITLE
B28G25_64 Nika

### PR DESCRIPTION
**Pull request by Nikgerya**
I create class with page object modules for 'InventoryManager10' account. Also in the 'features' you can find my feature file and scenario for my user story. To run my code and test the website I create step definition class named 'B28G25_64'

In configuration.properties file i just delete an extra space in on og the lines.

### CODE REVIEW

**Comment by Nikgerya** [comment]
My user story: As an Inventory manager, i want to access for all modules buttons

File: src/test/java/com/fidexio/pages/B28G25_64Page.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        listOfButtons.add(employeesModuleLinkButton);
+        listOfButtons.add(leavesModuleLinkButton);
+        listOfButtons.add(expensesModuleLinkButton);
+        listOfButtons.add(lunchModuleLinkButton);
+        listOfButtons.add(maintenanceModuleLinkButton);
+        listOfButtons.add(dashboardModuleLinkButton);
+        listOfButtons.add(discussModuleLinkButton);
+
+        return listOfButtons;
+    }
```
Comment (ManaalCydeo) [inline src/test/java/com/fidexio/pages/B28G25_64Page.java:91]: You could've just located the parent webElement for all modules then loop through if you needed to click to a specific one!

**Review by annadolganova3 — APPROVED** [review]
Well done!

File: src/test/java/com/fidexio/step_definitions/B28G25_64.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    */
+
+
+    @When("verify user is able go to Modules")
+    public void verifyUserIsAbleGoToModules() {
+
+       for (WebElement eachButton : buttonLocators.listOfButtons()) {
+            BrowserUtils.sleep(1);
+            Assert.assertTrue(eachButton.isEnabled());
```
Comment (ManaalCydeo) [inline src/test/java/com/fidexio/step_definitions/B28G25_64.java:98]: Nice job!!

**Review by ManaalCydeo — APPROVED** [review]

**Comment by Nikgerya** [comment]
Thank you!)

ср, 8 бер. 2023 р. о 17:56 Manaal ***@***.***> пише:

**Pull request merged**